tests/testapp/tests/test_worst_report_integrates_with_django_testrunner.py

```diff
@@ -23,37 +23,50 @@ def test_whatever_two(self):
         def test_slow_query(self):
             list(Group.objects.all())
 
-    return run_testcase_with_django_runner(SampleTestCase, nr_of_tests=3)
+    def get_packaged_runner_with_options(options=None):
+        options = options or {}
+        return RunnerTestCasePackage(SampleTestCase, nr_of_tests=3,
+                                     runner_options=options)
+    return get_packaged_runner_with_options
 
 
-def test_runner_has_worst_report_attribute(sample_test_results):
-    test_runner = sample_test_results['test_runner']
-    assert isinstance(test_runner.djpt_worst_report, WorstReport)
+def test_runner_has_worst_report_attribute(packaged_runner):
+    test_package = packaged_runner()
+    test_package.run()
+    assert isinstance(test_package.test_runner.djpt_worst_report, WorstReport)
 
 
-def test_report_is_printed_after_test_is_run(sample_test_results):
-    test_runner = sample_test_results['test_runner']
-    printed = sample_test_results['out']
-    assert printed.endswith(test_runner.djpt_worst_report.rendered())
+def test_report_is_printed_after_test_is_run(packaged_runner):
+    test_package = packaged_runner()
+    _result, output = test_package.run()
+    assert output.endswith(
+        test_package.test_runner.djpt_worst_report.rendered())
 
 
-def test_after_running_django_testcases_report_is_printed(sample_test_results):
-    whatever = get_report_value_for(sample_test_results, 'whatever', '')
+def test_no_report_is_printed_with_print_report_set_to_false(packaged_runner):
+    test_package = packaged_runner({"print_report": False})
+    _result, output = test_package.run()
+    assert output.find("Worst Performing Items") is -1, \
+        "Output was: {}".format(output)
```

Comment:
@belugame 

* py.test takes care of printing the involved local variables during test failure, thus the extra assertion failure message is not needed
* minor: `assert 'Worst Performing Items' not in output` feels more pythonic to me